app/views/alaveteli_pro/batch_request_authority_searches/_public_bodies.html.erb

```diff
@@ -1,8 +1,10 @@
 <ul>
   <% public_bodies = PublicBody.with_tag(category_tag) %>
-  <% cache [:batch_builder, locale, category_tag, public_bodies] do %>
+  <% key = rails5? ? public_bodies : public_bodies.maximum(:updated_at) %>
+  <% cache [:batch_builder, locale, category_tag, key] do %>
     <% public_bodies.select(&:is_requestable?).each do |public_body| %>
-      <% cache [:batch_builder, locale, category_tag, public_body] do %>
+      <% body_key = "public_bodies/#{public_body.id}-#{public_body.updated_at}" %>
```

Comment:
minor nitpick (2/2) -  if we use public_bodies(:updated_at) here, we get "false" in the cache key e.g. `school/29/2/15/23/4/2019/2/113/false/UTC/5d6745201cce2d42dca055b44c33923d`

if we use `public_bodies.updated_at.to_i`, we get `school/1556031749/da061a7420597d3b51959a58ba240c97`

Reply:
Also changed in 4c3c23c